packages/quantic/force-app/main/default/lwc/quanticDateFacet/quanticDateFacet.html

```diff
@@ -16,13 +16,13 @@
           <ul class="slds-has-dividers_around-space slds-m-vertical_xx-small">
             <template if:true={hasActiveValues}>
               <lightning-button
-                class="slds-p-horizontal_small facet__collapse"
+                class="facet__collapse"
```

Comment:
I just noticed that the `facet__collapse` class should not be set on this element. My bad.